opentech/public/news/templates/news/news_page.html

```diff
@@ -4,20 +4,33 @@
 {% block content %}
 <div class="wrapper wrapper--small">
 
-    <h1>{{ page.title }}</h1>
-
-    {% for author in page.authors.all %}
-        <p><a href="{% pageurl author.author %}">{{ author.author }}</a><p>
-    {% endfor %}
-
-    <p>{{ page.introduction }}</p>
+    <h5 class="heading heading--author">
+        {{ page.display_date|date:"D, Y-m-d H:i" }}
+        {% if page.authors.all %}
```

Comment:
I think this should be ok here, the QS is only small. For your reference: 
https://docs.djangoproject.com/en/2.0/ref/models/querysets/#when-querysets-are-evaluated
https://docs.djangoproject.com/en/2.0/ref/models/querysets/#django.db.models.query.QuerySet.exists

Reply:
cool, thanks Todd